### PR TITLE
Add panels

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
Add panel for the landing page.
## Note
* Kept same image as the top image for the project which the images are not available.
gett

### CODE REVIEW

**Review by jorgepiloto — CHANGES_REQUESTED** [review]
Thanks for implementing this @Revathyvenugopal162, looks good to me.

I see the panels still show a light background in dark mode. I guess we should adapt `ansys-sphinx-theme` to work with those before merging this.

**Comment by RobPasMue** [comment]
Talked to @Revathyvenugopal162 on a 1-to-1 call. We should implement better images for some of the projects + add a Getting Started section which should point to the main README of the project. She's working on that part :)

**Review by jorgepiloto — APPROVED** [review]

**Comment by jorgepiloto** [comment]
Just one last question. Why this documentation has a `_static/css` folder. Shouldn't all CSS files be inherited from `ansys-sphinx-theme`? 🤔 @RobPasMue @Revathyvenugopal162

**Review by MaxJPRey — APPROVED** [review]

File: doc/source/index.rst [hunk 1]
```diff
@@ -1,18 +1,91 @@
...
+            :link: https://fluentparametric.docs.pyansys.com/ 
+             
+            Pythonic interface to Ansys Fluent parametric workflows
+
+    .. grid-item::
+        .. card:: PyFluent-Visualization
+            :img-top: _static/assets/pyfluent.png
+            :link: https://fluentvisualization.docs.pyansys.com 
+             
+            Pythonic interface to visualize Ansys Fluent simulations using Python
```
Comment (MaxJPRey) [inline doc/source/index.rst:63]: Pythonic should be enough.
```suggestion
            Pythonic interface to visualize Ansys Fluent simulations
```

**Comment by MaxJPRey** [comment]
@Revathyvenugopal162 This looks really nice. Well done.

**Comment by MaxJPRey** [comment]
I did not check yet, but are we enforcing the card to have the same dimension whatever the content?
image

**Comment by MaxJPRey** [comment]
@Revathyvenugopal162 For AEDT I will try to get the same model but with an "E" field plotted.
But for now it is good as it is.

**Comment by Revathyvenugopal162** [comment]
> Just one last question. Why this documentation has a `_static/css` folder. Shouldn't all CSS files be inherited from `ansys-sphinx-theme`? 🤔 @RobPasMue @Revathyvenugopal162

@jorgepiloto At moment we don't have any customization for the  sphinx design in ansys-sphinx-theme. Since this repo specifically for landing, which uses sphinx panels, just a small modification for sphinx extension.

**Comment by Revathyvenugopal162** [comment]
> @Revathyvenugopal162 I did not check yet, but are we enforcing the cards to have the same dimension whatever the content? image
> 
> I think we should if possible.

The cards are having same dimensions . but the content inside makes the bottom border for the notes.

**Pull request merged**